app/fields/building_photo_field.rb

```diff
@@ -0,0 +1,7 @@
+require "administrate/field/base"
+
+class BuildingPhotoField < Administrate::Field::Base
+  def to_s
+	    data
+  end
```

Comment:
What will this look like if we do `building.photo.to_s`? Should it be something else, like the name of the file?

Reply:
building_photo_field has now been deleted. we're not using photos for buildings per cynthia.